### PR TITLE
Fix: porting link format

### PR DESCRIPTION
**Pull request by kotp**
Trivial link format correction to guidelines.

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]

**Review by wolf99 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -178,8 +178,7 @@ A problem must have a unique slug. This slug is used as
 
 ### In exercism/\<TRACK_ID\>
 
-* Do the same as when [porting an
-  exercise](https://github.com/exercism/legacy-docs/blob/main/you-can-help/implement-an-exercise-from-specification.md).
+* Do the same as when [porting an exercise][porting and exercise].
```
Comment (jiegillet) [inline CONTRIBUTING.md:181]: Typo:
```suggestion
* Do the same as when [porting an exercise][porting an exercise].
```
I'm also very wary of spaces in a keyword, I would prefer `porting-an-exercise`. All other links here follow that convention.

Comment (kotp) [reply CONTRIBUTING.md:181]: I prefer to not have the hyphens, as is seen in the formatting.  But if you want to suggest that change, please do.

**Review by sudhackar — APPROVED** [review]

**Comment by kotp** [comment]
@exercism/reviewers Waiting on one more approval.

**Review by glennj — APPROVED** [review]

**Comment by glennj** [comment]
oooh, I have power now, heh heh heh.

**Comment by kotp** [comment]
Thank you @jiegillet, @glennj, and, @wolf99.  This moment would not be possible without your help and guidance!

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sudhackar) [reply CONTRIBUTING.md]: @glennj Yeah It has been fixed. See the "outdated" above

**Pull request merged**